dictionaries/shell/src/legacy-words.txt

```diff
@@ -1,5 +1,6 @@
 alias
 allexport
+alnum
```

Comment:
Legacy Words isn't used for generating lists. It is just used in testing.

One of the other options might be better:
- `dictionaries/shell/src/command-words.txt` - for commands
- `dictionaries/shell/src/shell-sh-words.txt` - general shell related terms and keywords
- `dictionaries/shell/src/miscellaneous-words.txt` - things that do not fit in the other files.

Reply:
Ahh, I see, cool, changed.